src/actions/readMore.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const readMore = ID => dispatch => axios.get(`http://www.omdbapi.com/?apikey=fcbd49b5&i=${ID}`)
```

Comment:
Change name and types of action creator, for more understandable version (getSingeleMovie for example).